media-video/motion/motion-4.4.0.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7

```

Comment:
Always aim to use the latest available EAPI. Currently it's 8. A lot is determined by the eclass support, and it looks like these eclasses do support EAPI-8.